molgenis-settings/src/main/java/org/molgenis/settings/entity/EntitySettingsPackage.java

```diff
@@ -0,0 +1,36 @@
+package org.molgenis.settings.entity;
+
+import static java.util.Objects.requireNonNull;
+import static org.molgenis.data.system.model.RootSystemPackage.PACKAGE_SYSTEM;
+
+import org.molgenis.data.meta.SystemPackage;
+import org.molgenis.data.meta.model.PackageMetadata;
+import org.molgenis.data.system.model.RootSystemPackage;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+public class EntitySettingsPackage extends SystemPackage {
+  public static final String SIMPLE_NAME = "entity";
+  public static final String PACKAGE_ENTITY_SETTINGS =
```

Comment:
**suggestion:** Consider putting this package in the already existing settings package.

This will keep the settings organised.

Reply:
This is a different type of settings entity, the other entities assume a single row, this settings entity uses a row per entity. To make this clear i suggest to keep this entity in a separate package